### PR TITLE
showing_all_magazines

### PR DESCRIPTION
**Pull request by nbsven**

### CODE REVIEW

File: src/main/java/com/epam/magazinestore/controller/CatalogController.java [hunk 1]
```diff
@@ -1,17 +1,30 @@
...
 
 @Controller
 public class CatalogController {
 
+    @Autowired
+    MagazineService magazineService;
+
     @GetMapping(value = CATALOG)
-    public String catalog() {
+    public String catalog(Model model) {
```
Comment (AntonKiselyov) [inline src/main/java/com/epam/magazinestore/controller/CatalogController.java]: Controller mustn't contain any logic, replace logic to service class, please

Comment (Jebsuz) [reply src/main/java/com/epam/magazinestore/controller/CatalogController.java]: i don't see any business logic here, what's wrong?

File: src/main/java/com/epam/magazinestore/controller/CatalogController.java [hunk 2]
```diff
@@ -1,17 +1,30 @@
...
+        List<Magazine> magazines = magazineService.getAvailableMagazines(0, Integer.MAX_VALUE);
+        System.out.println(magazines.size());
```
Comment (Jebsuz) [inline src/main/java/com/epam/magazinestore/controller/CatalogController.java]: either remove it or use lombok's annotation `@Slf4j` and log it via `log.info` or even `log.debug`

File: src/main/java/com/epam/magazinestore/controller/CatalogController.java [hunk 3]
```diff
@@ -1,17 +1,30 @@
...
+
 import static com.epam.magazinestore.util.JSPTitleUtil.CATALOG_JSP;
 import static com.epam.magazinestore.util.PathUtils.CATALOG;
 
 
 @Controller
 public class CatalogController {
 
+    @Autowired
+    MagazineService magazineService;
```
Comment (Jebsuz) [inline src/main/java/com/epam/magazinestore/controller/CatalogController.java]: This mistake shouldn't happen. We already talked about it, use constructor injection

File: src/main/java/com/epam/magazinestore/controller/CatalogController.java (hunk 1 above)
Comment (AntonKiselyov) [reply src/main/java/com/epam/magazinestore/controller/CatalogController.java]: Maybe I see not there. Sorry, it's Ok here

**Pull request merged**